Framework/CurveFitting/test/Algorithms/RefinePowderInstrumentParameters3Test.h

```diff
@@ -204,289 +470,75 @@ class RefinePowderInstrumentParameters3Test : public CxxTest::TestSuite {
 
     return;
   }
+};
 
-  //----------------------------------------------------------------------------------------------
-  /** Generate a table workspace for holding instrument parameters for POWGEN's
-   * bank 1
-    */
-  TableWorkspace_sptr generateInstrumentProfileTableBank1() {
-    DataObjects::TableWorkspace *tablews = new DataObjects::TableWorkspace();
-    DataObjects::TableWorkspace_sptr geomws =
-        DataObjects::TableWorkspace_sptr(tablews);
-
-    tablews->addColumn("str", "Name");
-    tablews->addColumn("double", "Value");
-    tablews->addColumn("str", "FitOrTie");
-    tablews->addColumn("double", "Min");
-    tablews->addColumn("double", "Max");
-    tablews->addColumn("double", "StepSize");
-
-    API::TableRow newrow = geomws->appendRow();
-    newrow << "Dtt1" << 22778.3 << "f" << 0.0 << 1.0E20 << 1.0;
-
-    newrow = geomws->appendRow();
-    newrow << "Dtt1t" << 22747.4 << "t" << 0.0 << 1.0E20 << 1.0;
-
-    newrow = geomws->appendRow();
-    newrow << "Dtt2" << 0.0 << "t" << 0.0 << 1.0E20 << 1.0;
-
-    newrow = geomws->appendRow();
-    newrow << "Dtt2t" << 0.3 << "t" << -10000.0 << 100000.0 << 1.0;
-
-    newrow = geomws->appendRow();
-    newrow << "Tcross" << 0.356 << "t" << 0.0 << 1000.0 << 1.0;
-
-    newrow = geomws->appendRow();
-    newrow << "Width" << 1.1072 << "f" << 0.0 << 1000.0 << 1.0;
-
-    newrow = geomws->appendRow();
-    newrow << "Zero" << 0.0 << "f" << -10000.0 << 10000.0 << 1.0;
-
-    newrow = geomws->appendRow();
-    newrow << "Zerot" << 90.7 << "t" << -10000.0 << 10000.0 << 1.0;
-
-    return geomws;
-  }
-
-  //----------------------------------------------------------------------------------------------
-  /** Generate a table workspace for holding instrument parameters
-    */
-  TableWorkspace_sptr generateInstrumentProfileTable(int bankid) {
-    // 1. Import
-    vector<string> colnames;
-    vector<vector<string>> strparams;
-
-    if (bankid == 1) {
-      string filename(
-          "/home/wzz/Mantid/Code/debug/MyTestData/bank1profile.txt");
-      importTableTextFile(filename, strparams, colnames, 6);
-    } else {
-      throw runtime_error("generateInstrumentProfile supports bank 1 only.");
-    }
+class RefinePowderInstrumentParameters3TestPerformance
+    : public CxxTest::TestSuite {
```

Comment:
@raquelalvarezbanos Someone else had the same problem recently: cxxtest beaks if the class declaration spans 2 lines (but clang-format forces it to two lines). In the end we used this trick that saves a few characters, see if that fits:

```cpp
using CxxTest::TestSuite;
class RefinePowderInstrumentParameters3TestPerformance : public TestSuite {
```